src/i18n.tsx

```diff
@@ -1,8 +1,9 @@
+import { ReactNode, useCallback } from 'react'
+
 import { SupportedLocale } from 'legacy/constants/locales'
 import { initialLocale, useActiveLocale } from 'legacy/hooks/useActiveLocale'
-import { dynamicActivate, Provider } from 'lib/i18n'
-import { ReactNode, useCallback } from 'react'
 import { useUserLocaleManager } from 'legacy/state/user/hooks'
+import { dynamicActivate, Provider } from 'lib/i18n'
```

Comment:
For example, here, legacy shouldn't belong to the same group as lib

Reply:
Agree, fixed

Reply:
Agree, fixed